src/components/PhotoBox.js

```diff
@@ -18,15 +23,26 @@ export function linkPath(item) {
 
 function PhotoBox(props) {
   const { item } = props;
+  const styles = {
+    linkWrapper: {
+      cursor: 'pointer'
+    },
+    title: {
+      display: 'inline-block',
+      marginTop: '12px'
+    }
+  };
 
   return (
     <article aria-labelledby="grid1" className="photo-box">
-      <a>
-        <img alt={item._source.title.primary[0]} src={imagePath(item)} />
-      </a>
-      <h4 id="grid1">
```

Comment:
@kdid Was that `id="grid1"` being used for anything, or did it just get copied over?   Didn't look like it was being referenced anywhere when I did a search.

Reply:
I think you're right - probably copied over.